_includes/manuals/nightly/3.3.1_rpm_packages.md

```diff
@@ -57,10 +25,9 @@ Under each repo are directories for each distribution, then each architecture.
 
 To set up the repository, foreman-release packages are provided which add a repo definition to `/etc/yum.repos.d`.  Install the appropriate release RPM from these lists:
```

Comment:
It's no longer really a list, is it? Perhaps we should also mention the module enablement:
```suggestion
To set up the repository, foreman-release packages are provided which add a repo definition to `/etc/yum.repos.d`.  Install the release RPM and enable the module:
```